solidity/test/BancorConverter.js

```diff
@@ -55,8 +55,8 @@ async function initConverter(accounts, activate, maxConversionFee = 0) {
     return converter;
 }
 
-function verifyReserve(reserve, virtualBalance, weight, isSet) {
-    assert.equal(reserve[0], virtualBalance);
+function verifyReserve(reserve, balance, weight, isSet) {
```

Comment:
do we actually need to test the balance here? isn't it deprecated in the reserve struct?

Reply:
@yudilevi : No, it actually isn't. We've replaced the (not quite used) virtual balance with the (now very much in use) real balance.

Reply:
ahh right